spec/models/comment_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+Rspec.describe Comment, type: :model do
+  test_user = User.create(name: 'yankee', bio: 'Teacher from Abuja')
+  test_post = Post.create(author: test_user, title: 'testing', text: 'thanks for testing this')
+  subject { Comment.new(author: test_user, post: test_post, text: 'hey thanks for having me ') }
+
+  it 'updates comment counter post' do
+    expect(subject.post.comments_counter).to be 1
+  end
+  it ' user comment should be David' do
+    expect(subject.post.author.name).to be 'yankee'
```

Comment:
- [ ] This test fails because you are using `be`. When you use `be` Rspec tryes to compare the object instance, not the actual string value, so even if the values are the same it would fail. Kindly change to something like this and use the `eq` matcher.
  ```
  expect(subject.post.author.name).to eq 'yankee'
  ```
 
  ![yankee](https://user-images.githubusercontent.com/54848363/190686228-c7ac6395-61a2-42a9-aea3-98b15e7b3582.jpg)